google-cloud-bigquery/src/test/java/com/google/cloud/bigquery/it/ITBigQueryTest.java

```diff
@@ -707,6 +707,60 @@ public void testCreateTableWithRangePartitioning() {
     }
   }
 
+  @Test
+  public void testJsonType() throws InterruptedException {
+    String tableName = "test_create_table_jsontype";
+    TableId tableId = TableId.of(DATASET, tableName);
+    Schema schema = Schema.of(Field.of("jsonField", StandardSQLTypeName.JSON));
+    StandardTableDefinition standardTableDefinition = StandardTableDefinition.of(schema);
+    try {
+      // Create a table with a JSON column
+      Table createdTable = bigquery.create(TableInfo.of(tableId, standardTableDefinition));
+      assertNotNull(createdTable);
+
+      // Insert data into JSON column
+      Map<String, Object> jsonRow = new HashMap<>();
+      jsonRow.put("jsonField", "{\"class\" : {\"students\" : [{\"name\" : \"Jane\"}]}}");
```

Comment:
How about inserting multiple rows each with a different name/id

e.g.
      // Insert data into JSON column
      Map<String, Object> jsonRow1 = Collections.singletonMap("jsonField", "{\"student\" : {\"name\" : \"Jane\", \"id\": 10}}");
      Map<String, Object> jsonRow2 = Collections.singletonMap("jsonField", "{\"student\" : {\"name\" : \"Joy\", \"id\": 11}}");
      Map<String, Object> jsonRow3 = Collections.singletonMap("jsonField", "{\"student\" : {\"name\" : \"Alice\", \"id\": 12}}");
      Map<String, Object> jsonRow4 = Collections.singletonMap("jsonField", "{\"student\" : {\"name\" : \"Bijoy\", \"id\": 13}}");

      InsertAllRequest request  = InsertAllRequest.newBuilder(tableId).addRow(jsonRow1).addRow(jsonRow2).addRow(jsonRow3).addRow(jsonRow4).build();

Reply:
Updating test case to insert all these rows (it passed).